src/utils/api/api.js

```diff
@@ -1,5 +1,6 @@
 import qs from 'qs';
 import { ApiError } from './errors';
+import * as apiConstants from '../../constants/api';
```

Comment:
I agree with @justafish that this makes everything further down much harder to parse.

Some of these strings are only used once, so moving them to a 'constants' file does not really make sense. Things that are used over and over, like `_format=json` or `application/vnd.api+json` perhaps make sense, but those could also just be stored in this file.